packages/strapi-plugin-sentry/config/functions/bootstrap.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = async () => {
+  // Initialize the Sentry service exposed by this plugin
+  const { sentry } = strapi.plugins.sentry.services;
+  sentry.init();
+
+  // Only send errors to Sentry if a valid DSN was entered and the plugin isn't disabled
+  const shouldSendSentryEvents = sentry.isReady;
+
+  // Create a middleware to intercept API errors
+  strapi.app.use(async (ctx, next) => {
+    try {
+      await next();
+    } catch (error) {
+      if (shouldSendSentryEvents) {
+        sentry.sendError(error, (scope, sentryInstance) => {
```

Comment:
don't need to check if it is ready this can stay inside the service as you already did.